wordpress-mobile/WordPress-iOS
# Add gravatar upload to signup epilogue

Refs #8109

This adds support for uploading a new gravatar at the end of the signup process. This is one of the last remaining issues for the Signup Master Issue.

![gravatar upload animation](https://user-images.githubusercontent.com/517257/41699953-8c770a96-74e3-11e8-8162-b700e8528af5.gif)

To test:
- signup with a new account
- ensure you can upload a new gravatar image
- ensure that the me tab's gravatar uploading still works

Note: The app seems to cache gravatar images pretty aggressively. Best bet to double check that the upload works is to check it somewhere else, like the web or another instance of the app.
